Application/SchedulingLib/LocalScheduler.cs

```diff
@@ -13,7 +13,7 @@ public LocalScheduler()
             context.Database.Migrate();
         }
 
-        public void NewInstructor(Instructor instructor)
+        public void NewInstructor(DatabaseModels instructor)
```

Comment:
This should still be Instructor (I don't think there is a class called DatabaseModels)

Reply:
```suggestion
        public void NewInstructor(Instructor instructor)
```